packages/core/admin/admin/src/content-manager/components/BlocksEditor/tests/index.test.js

```diff
@@ -5,12 +5,15 @@ import { render, screen } from '@testing-library/react';
 import { IntlProvider } from 'react-intl';
 
 import Blocks from '../index';
+import { blocksData } from '../tempSchema';
```

Comment:
I think blocksData should be moved into the tests folder since it's only used here